packages/contracts-sdk/src/lib/contracts-sdk.ts

```diff
@@ -986,6 +992,8 @@ export class LitContracts {
       'https://lit-general-worker.getlit.dev/habanero-contract-addresses';
     const DATIL_DEV_API =
       'https://lit-general-worker.getlit.dev/datil-dev/contracts';
+    const DATIL_TEST_API =
+      'https://staging.apis.getlit.dev/datil-test/contracts';
```

Comment:
Not necessarily a blocker, but I am curious why just this entry is using `staging.apis` when all the rest use general worker - do we need to ship another PR to update this URL at some point later, and if so, when do we know we can/should do that?

Reply:
Yea, so the `staging.apis` is actually from a staging worker. I've been running it a few days to make sure that it doesn't break. I've now merged it to master https://github.com/LIT-Protocol/lit-apps/pull/22, so I will update this to use the general worker instead.